src/main/java/org/cascadebot/cascadebot/commands/developer/GuildInfoSubCommand.java

```diff
@@ -46,18 +44,20 @@ public void onCommand(Member sender, CommandContext context) {
 
 
         GuildData finalDataForList = dataForList;
+        /*
         String flags = Arrays.stream(Flag.values())
                              .map(flag -> FormatUtils.formatEnum(flag) + " - " +
                                      (finalDataForList.isFlagEnabled(flag) ? UnicodeConstants.TICK : UnicodeConstants.RED_CROSS))
                              .collect(Collectors.joining("\n"));
+         */
```

Comment:
Probably best to remove this completely but also add a section on what tier the guild has.